### DISCUSSION TITLE
COMPOSER REQUIRE WEBAPP

### DISCUSSION THREAD

**Discussion by irengeBirindwaGloire** [curl/curl#13406] [General]
Hi team, I have a problem that seems a bit complicated to me because I can't understand the origin when I run the COMPOSER REQUIRE WEBAPP command. Here's the error :

 Failed to execute git clone --mirror -- https://ghp...HDE:x-oauth-basic@github.com/doctrine/dbal.git

**Comment by bagder** [comment]
Did you perhaps submit this question in the wrong place? I don't see how this question is at all related to curl or libcurl...
